### ISSUE TITLE
CMM results by macOS 14 don't match other platform results

### ISSUE THREAD

**Issue by ktbolt** [SimVascular/svFSIplus#137]
### Description

On macOS 14.0 Sonoma the `test_cmm_3d_pipe` results in field Stress differ by more than rtol=1e-12 in 11901 out of 11988 results. Max. abs. difference is 3.5e-08.

### Reproduction

Run the `test_cmm_3d_pipe` test.

### Expected behavior

The `test_cmm_3d_pipe` test results should match those computed on other platforms within rtol=1e-12.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct and Contributing Guidelines

**Comment by mrp089** [comment]
I just ran the tests again on my M2 macOS Sonoma 14.2.1. Unfortunately, it still fails for 1, 3, and 4 procs.

It fails for step `2b-prestress` in `test_pipe_3d`. I modified the fields on this line to see for what it fails:
https://github.com/SimVascular/svFSIplus/blob/6b841e80b54aca877c9592d145a0692b473dd6c9/tests/test_cmm.py#L24
Turns out It fails for `Stress` but passes for `Displacement`.

**Comment by ktbolt** [comment]
I'm doing some tests on MacOS Catalina and have found that the the prestress option `<Initialize> prestress </Initialize>
` was not being processed correctly for the `phys_CMM` equation type, causing `Stress` values to not be written to VTK results files.

**Comment by mrp089** [comment]
In case you didn't see it, please have a look at #173. This might be related since the `phys_CMM` was the only test that actually tested `Stress`.

**Comment by ktbolt** [comment]
@mrp089 So it seems that only the first field `Displacement` was tested ,`Stress` was never tested?

**Comment by mrp089** [comment]
[This list](https://github.com/SimVascular/svFSIplus/issues/173#issuecomment-1909248168) contains what we actually tested so far. The `cmm` test is the only one that **did** test `Stress`, in this step:
https://github.com/SimVascular/svFSIplus/blob/372851e39e163cfcb4462c2a5ca98b06337dc058/tests/test_cmm.py#L22-L24

There were other [discrepancies](https://github.com/SimVascular/svFSIplus/issues/173#issuecomment-1910904068) for different numbers of CPUs and different operation systems in `Stress` results when enabling the tests. So maybe this points to a `Stress` post-processing bug?

**Comment by ktbolt** [comment]
I did not see any large discrepancies when comparing C++ and Fortran results in `svFSI-Tests` but let me have a look and see if I can find anything suspicious.

**Comment by ktbolt** [comment]
@mrp089 I have not found any problems in the `Stress` post-processing. 

Comparing `svFSI-Tests/05-struct/01-block-compression` results I see
```
Data name: Stress   Type: 11
  num_points: 35937
  num_comp: 6
  General double nodal data
  Values 1 min:  -5.83156e+06, -5.83156e+06, -6.31574e+06, -587374, -476119, -476119
  Values 2 min:  -5.83156e+06, -5.83156e+06, -6.31575e+06, -587375, -476120, -476120
   
  Values 1 max:  1.31777e+06, 1.31777e+06, 897277, 61747.6, 1.11555e+06, 1.11555e+06
  Values 2 max:  1.31777e+06, 1.31777e+06, 897279, 61747.7, 1.11555e+06, 1.11555e+06
   
  Max diffs:  3.43034, 3.43039, 7.02925, 1.17859, 2.21608, 2.21608
  Max pdiffs:  4.79813e-05, 4.79819e-05, 9.74523e-05, 0.000181567, 0.00013923, 0.00013923
  **** Data does not match for 35937 entries. Max diff: 1007.03
```

The max/mins for each Stress component agree well and the differences between values are relatively small.

**Comment by mrp089** [comment]
I think this is closed by #175. In all test cases, I tightened the nonlinear and linear tolerances and set the tolerance of `Stress` to `1e-4`. The reference solutions (which I computed on macOS 14) pass on all test machines.
